OJS.Workers.ExecutionStrategies/Python/PythonUnitTestsExecutionStrategy.cs

```diff
@@ -90,6 +99,11 @@ protected override string SaveCodeToTempFile<TInput>(IExecutionContext<TInput> e
 
         private string GetTestCodeClassName(TestsInputModel testsInput)
         {
+            if (string.IsNullOrWhiteSpace(testsInput.TaskSkeletonAsString))
+            {
+                throw new ArgumentException(SolutionSkeletonMissingErrorMessage);
+            }
+
             var className = Regex.Match(testsInput.TaskSkeletonAsString, ClassNameInSkeletonRegexPattern)
```

Comment:
use testsInput.TaskSkeletonAsString ?? string.Empty instead of raising different exception